extensions/EXT/EGL_EXT_device_query_vendor.txt

```diff
@@ -0,0 +1,77 @@
+Name
+
+    EXT_device_query_vendor
```

Comment:
"query_vendor" doesn't really describe this extension, since it adds a query for both the vendor and device ID's. Also, "vendor" sometimes means "driver" in GL/EGL.

Maybe something like `EGL_EXT_device_query_pci`?

Reply:
Good point, I'll rename it :+1:

`EGL_EXT_device_query_pci_id` perhaps?

Reply:
Good point.  `EGL_EXT_device_query_pci_id` sounds very clearly descriptive to me.